test/integration/KarmaTestRunner.it.ts

```diff
@@ -13,7 +13,19 @@ describe('KarmaTestRunner', function () {
   let expectToHaveFailedTests = (result: RunResult, expectedFailureMessages: string[]) => {
     const actualFailedTests = result.tests.filter(t => t.status === TestStatus.Failed);
     expect(actualFailedTests).to.have.length(expectedFailureMessages.length);
-    actualFailedTests.forEach(failedTest => expect(failedTest.failureMessages[0]).to.contain(expectedFailureMessages.shift()));

```

Comment:
I don't like test code being so bloated just because we force strict nulls in production code. We should either ignore strict null-checks here or do an `any` cast